### PR TITLE
Release rethinkdb 2.4.1

### PR DESCRIPTION
**Pull request by gabor-boros**
Hello,

We would like to push our new version 2.4.1 to docker hub.

### CODE REVIEW

**Comment by gabor-boros** [comment]
Could you please restart the two failing builds? The issue was on the download server and fixed now.

File: library/rethinkdb [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-Tags: 2.4.0-buster-slim, 2.4-buster-slim, 2-buster-slim, buster-slim, 2.4.0, 2.4, 2, latest
+Tags: 2.4.1-buster-slim, 2.4-buster-slim, 2-buster-slim, buster-slim, 2.4.1, 2.4, 2, latest
+Directory: buster/2.4.1
+GitCommit: 5bccd499742f6f05e7ddc970842a1763bdd99cfa
+
+Tags: 2.4.1-centos, 2.4-centos, 2-centos, centos
+Directory: centos8/2.4.1
+GitCommit: 5bccd499742f6f05e7ddc970842a1763bdd99cfa
+
+Tags: 2.4.0-buster-slim, 2.4.0
```
Comment (tianon) [inline library/rethinkdb:12]: Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (tianon) [reply library/rethinkdb:12]: Friendly ping :wink: :heart:

Comment (marchaos) [reply library/rethinkdb:12]: @gabor-boros Not sure if you saw this comment?

Comment (gabor-boros) [reply library/rethinkdb:12]: @marchaos Sorry I clearly missed this comment.

@tianon I made a task for myself for the next week to open a PR removing those tags.

Comment (tianon) [reply library/rethinkdb:12]: :eyes:

Comment (gabor-boros) [reply library/rethinkdb:12]: Oh! Sorry, I totally missed this one! I'll be back in ~2 weeks and check it for sure!

Comment (gabor-boros) [reply library/rethinkdb:12]: Shame on me @tianon, the PR is: https://github.com/docker-library/official-images/pull/11149

Comment (tianon) [reply library/rethinkdb:12]: It's all good -- thanks for getting back to it! :smile: :heart:

**Pull request merged**